contents/tutorials/framer-analytics.md

```diff
@@ -0,0 +1,112 @@
+
+```html
+<script>
+    !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.async=!0,p.src=s.api_host+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled onFeatureFlags getFeatureFlag getFeatureFlagPayload reloadFeatureFlags group updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures getActiveMatchingSurveys getSurveys".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
+    posthog.init('<ph_project_api_key>',{api_host:'<ph_instance_address>'})
+</script>
+```
+
+With the JavaScript snippet copied, go to your Framer project and click the gear in the top right to go to your site settings. If you haven’t already, sign up for the "Mini" site plan. This enables you to add custom code.
+
+Once on a paid plan, go to the "General" tab in site settings and scroll down to the "Custom Code" section. Under "End of `<head>` tag", paste your PostHog JavaScript snippet here. Make sure to press "Save" next to the "Custom Code" heading. 
+
+![Script](../images/tutorials/framer-analytics/script.png)
+
+Once you publish and go to your site, PostHog starts to autocapture events like pageviews, button clicks, and inputs for your site.
+
+![Autocapture](../images/tutorials/framer-analytics/autocapture.png)
+
+### Enabling session replays
+
+To enable session replays, go to [your project settings](https://app.posthog.com/project/settings#recordings), scroll down to "Recordings," and enable "Record user sessions." PostHog then starts to record sessions of your users, so you can see all the details of how they use your site.
+
+## Adding a custom event
+
+You can add [custom code components](https://www.framer.com/developers/#code-components) to Framer and because of that, we can set up [custom event capture](/docs/libraries/js#capturing-events). We will create a basic button that captures a `clicked homepage button`  and sets a `clicked_homepage_button` person property to `true`.
+
+To do this, go to the "Assets" tab in the top left of your project. Next to the "Code" tab, click the plus icon to create a code file. Name the file `CaptureButton`, set it as a "New component" and press "Create."
```

Comment:
video or screenshot could be nice

Reply:
It's included in the video below so I'll just leave as is.